tfhe-zk-pok/src/proofs/pke_v2.rs

```diff
@@ -2758,119 +2579,28 @@ mod tests {
 
     #[test]
     fn test_proof_compression() {
-        let d = 2048;
-        let k = 320;
-        let B = 1048576;
-        let q = 0;
-        let t = 1024;
-
-        let msbs_zero_padding_bit_count = 1;
-        let effective_cleartext_t = t >> msbs_zero_padding_bit_count;
-
-        let delta = {
-            let q = if q == 0 { 1i128 << 64 } else { q as i128 };
-            (q / t as i128) as u64
-        };
+        let PkeTestParameters {
+            d,
+            k,
+            B,
```

Comment:
we don't recompute the upper bound of the std dev with the formula I gave you for TUniform stuff ?

Reply:
If I understand correctly the formula is to convert from a gaussian stddev, but here we already have a TUniform value

Reply:
actually the fomula should give an equivalent std dev for the TUniform that can safely be used in places where a gaussian is expected, not sure it is the exact std dev for a TUniform distribution however, guess we could check that formula